test_pull_requests

```diff
@@ -124,12 +124,37 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+# TODO: create a :state to "content" mapping for composing new bot
+# messages
+PR_STATES = {
+  /^Evaluating/ => :evaluating,
```

Comment:
I'm not sure I'd call it a "hodgepodge", but I agree that this isn't very
clean. The parsing rules are half in the map and half in the method that
uses the map. I'd much prefer that to map string literals to states.

As far as not being states for the same thing, do you mean the `ACTION_*`
keys don't belong with the others? I'll try to find a way to break those
out.

On Fri, Aug 12, 2016 at 8:46 AM, Dan McPherson <notifications@github.com>
wrote:

> In test_pull_requests
> <https://github.com/openshift/test-pull-requests/pull/31#discussion_r74584268>
> :
>
> > @@ -124,12 +124,37 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
> >  GITHUB_API_BASE_URL = 'https://api.github.com'
> >  GITHUB_BASE_URL = 'https://github.com'
> >
> > +# TODO: create a :state to "content" mapping for composing new bot
> > +# messages
> > +PR_STATES = {
> > +  /^Evaluating/ => :evaluating,
>
> I don't like this hodgepodge of things in a map. They are not all states
> for the same thing. I also don't love the abstraction being so fuzzy.
> Meaning checking for the beginning of the string but not the end. If we are
> going to abstract it, you need to be more precise. Otherwise the details of
> the fuzziness will be forgotten from the callers.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/test-pull-requests/pull/31/files/f6be253ba3714952ed04378e0054454993bcca3b#r74584268>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABd0FCTVVtKR-4GGiwa7g-5CFvU6Lb69ks5qfGsIgaJpZM4JcBHJ>
> .
>